bitnami/airflow/templates/_git_helpers.tpl

```diff
@@ -83,6 +83,8 @@ Returns the init container that will clone repositories files from a given list
     - /bin/bash
     - -ec
     - |
+        . /opt/bitnami/scripts/git/entrypoint.sh || true
```

Comment:
Could you explain the `|| true`?

Reply:
It is just to prevent the command to fail if the container doesn't contain that file. That may happen if someone is using an old version of the Bitnami container or another container that is not maintained by us.

Maybe a more explicit check would be easy to understand:

```suggestion
        [[ -f "/opt/bitnami/scripts/git/entrypoint.sh" ]] && source "/opt/bitnami/scripts/git/entrypoint.sh"
```

Do you agree?

Reply:
I like that approach better.